hack/install-apiserver/install-apiserver.sh

```diff
@@ -0,0 +1,97 @@
+mv ${apiserverConfigDir}/tls.crt ${apiserverConfigDir}/serving.crt
+mv ${apiserverConfigDir}/tls.key ${apiserverConfigDir}/serving.key
+
+# 5.  Installer harvests SA token
+saToken=$(oc -n ${targetNamespace} sa get-token apiserver)
+
+# 6.  Installer creates SA kubeconfig
+# TODO do this a LOT better
+# start with admin.kubeconfig
+cp ${masterConfigDir}/admin.kubeconfig ${apiserverConfigDir}/kubeconfig
+# remove all users
+oc --config=${apiserverConfigDir}/kubeconfig config unset users
+# set the service account token
+configContext=$(oc --config=${apiserverConfigDir}/kubeconfig config current-context)
+oc --config=${apiserverConfigDir}/kubeconfig config set-credentials serviceaccount --token=${saToken}
+oc --config=${apiserverConfigDir}/kubeconfig config set-context ${configContext} --user=serviceaccount
+
+# 7.  Installer binds “known” roles to SA user
+# TODO remove this bit once we bootstrap these roles
+oc create -f hack/install-apiserver/prestart-resources.yaml || true
+
+oadm policy add-cluster-role-to-user system:auth-delegator -n ${targetNamespace} -z apiserver
+oc create policybinding kube-system -n kube-system
+oadm policy add-role-to-user extension-apiserver-authentication-reader -n kube-system --role-namespace=kube-system system:serviceaccount:${targetNamespace}:apiserver
+
+
+# 8.  Installer provides the following information to all templates.
+cp ${masterConfigDir}/ca.crt ${apiserverConfigDir}/etcd-ca.crt
+cp ${masterConfigDir}/ca-bundle.crt ${apiserverConfigDir}/client-ca.crt
+cp ${masterConfigDir}/master.etcd-client.crt ${apiserverConfigDir}/etcd-write.crt
+cp ${masterConfigDir}/master.etcd-client.key ${apiserverConfigDir}/etcd-write.key
+templateArgs="ETCD_URL=https://10.13.137.230:4001"
+templateArgs="${templateArgs} CLIENT_CA=/etcd/apiserver-config/client-ca.crt"
+templateArgs="${templateArgs} ETCD_CA=/etcd/apiserver-config/etcd-ca.crt"
+templateArgs="${templateArgs} ETCD_WRITE_CRT=/etcd/apiserver-config/etcd-write.crt"
+templateArgs="${templateArgs} ETCD_WRITE_KEY=/etcd/apiserver-config/etcd-write.key"
+templateArgs="${templateArgs} SERVING_CRT=/etcd/apiserver-config/serving.crt"
+templateArgs="${templateArgs} SERVING_KEY=/etcd/apiserver-config/serving.key"
+templateArgs="${templateArgs} KUBECONFIG=/etcd/apiserver-config/kubeconfig"
+templateArgs="${templateArgs} NAMESPACE=${targetNamespace}"
+templateArgs="${templateArgs} CONFIG_DIR=${apiserverConfigDir}"
+templateArgs="${templateArgs} CONFIG_DIR_MOUNT=/etcd/apiserver-config"
+
+# 9.  Installer processes and creates the pre-create template - this is the template that provides things like clusterroles, clusterrolebindings, random weird stuff.
+# nothing for wardle
+
+# 10. Installer processes the static pod template - this is a template that ONLY creates a static pod.
+oc process -f hack/install-apiserver/kube-wardle-pod-template.yaml ${templateArgs} | jq .items[0] > ${nodeManifestDir}/${targetNamespace}.yaml
```

Comment:
is this supposed to create this resource?  right now it's just writing the pod definition to a file...

also i see a reference to 13 steps but only 10 here... did it get simplified or did something get truncated?